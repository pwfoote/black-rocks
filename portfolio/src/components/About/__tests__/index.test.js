import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from '..';


afterEach(cleanup);

describe('Nav component', () => {
    it('renders', () => {
      render();
    })
  
    it('matches snapshot DOM node structure', () => {
      const { asFragment } = render();
  
      expect(asFragment()).toMatchSnapshot();
    });
  })