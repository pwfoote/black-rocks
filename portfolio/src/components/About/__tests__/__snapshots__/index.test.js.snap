// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component matches snapshot DOM node structure 1`] = `<DocumentFragment />`;
